apps/web/e2e/navigation-tests/electron-navigation.spec.ts

```
@@ -0,0 +1,32 @@
+
+import { test, expect, _electron } from '@playwright/test';
+
+test.describe('Electron Navigation', () => {
+  let electronApp;
```

Biome (2.1.2)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)